<launch>
    <node name="display_image_1" pkg="phobos_rover_vision" type="display_image" ns="rover">
        <param name="camera_name" value="camera1"/>
        <param name="width" value="320"/>
        <param name="height" value="240"/>
        <param name="x" value="320"/>
        <param name="y" value="0"/>
        <param name="image_transport" value="theora"/>
    </node>

    <node name="display_image_2" pkg="phobos_rover_vision" type="display_image" ns="rover">
        <param name="camera_name" value="camera2"/>
        <param name="width" value="320"/>
        <param name="height" value="240"/>
        <param name="x" value="320"/>
        <param name="y" value="240"/>
        <param name="image_transport" value="theora"/>
    </node>

    <node name="display_image_3" pkg="phobos_rover_vision" type="display_image" ns="rover">
        <param name="camera_name" value="camera3"/>
        <param name="width" value="320"/>
        <param name="height" value="240"/>
        <param name="x" value="0"/>
        <param name="y" value="0"/>
        <param name="image_transport" value="theora"/>
    </node>

    <node name="display_image_4" pkg="phobos_rover_vision" type="display_image" ns="rover">
        <param name="camera_name" value="camera4"/>
        <param name="width" value="320"/>
        <param name="height" value="240"/>
        <param name="x" value="0"/>
        <param name="y" value="240"/>
        <param name="image_transport" value="theora"/>
    </node>
</launch>
